.circleci/config.yml

```diff
@@ -71,8 +71,6 @@ jobs:
   push-docker:
     docker:
       - image: circleci/buildpack-deps:stable
```

Comment:
Can we also change this to `circleci/golang:1.14` ? This breaks the docker build phase because it doesn't have permissions to `/go` directory.

Reply:
because we were using a path in the executor that point to `/go` for this build there is nothing related to go build the docker build just download the binary and unpack

Reply:
in this new image the /go does not exist, only exist in the golang image